src/main/java/me/squidxtv/frameui/api/ScreenRegistry.java

```diff
@@ -12,6 +13,12 @@
  */
 public interface ScreenRegistry {
 
+    /**
+     * @param plugin the screen's plugin
+     * @return {@link ScreenBuilder} a class for easy and fluent creating of a new screen
+     */
```

Comment:
maybe for consistency add a general javadocs

Reply:
what do you mean by the General javadocs?

Reply:
The current documentation only uses `@param` and `@return` to describe its functionality, but there should be at least a sentence above it that describes it as well. Like:
```java
    /**
     * Creates a builder that ...
     * @param plugin the screen's plugin
     * @return {@link ScreenBuilder} a class for easy and fluent creating of a new screen
     */
```
or something like that

Reply:
done

Reply:
done